ttnn/ttnn/operations/conv2d.py

```diff
@@ -673,8 +670,8 @@ def conv2d(
             compute_kernel_config = ttnn.WormholeComputeKernelConfig(
                 math_fidelity=conv_config.math_fidelity,
                 math_approx_mode=conv_config.math_approx_mode_enabled,
-                fp32_dest_acc_en=conv_config.fp32_dest_acc_en_enabled,
-                packer_l1_acc=conv_config.packer_l1_acc_enabled,
```

Comment:
Hmm, how was this not erroring out earlier.

Reply:
The only tests that had debug mode enabled were for only for gray skull.